README.rst

```diff
@@ -35,7 +64,14 @@ co-refinements with other techniques.
 For more information about the diffpy.srfit library, see the users manual at
 http://diffpy.github.io/diffpy.srfit.
 
-REQUIREMENTS
+Citation
+--------
+
+If you use diffpy.srfit in a scientific publication, we would like you to cite this package as
+
+        diffpy.srfit Package, https://github.com/diffpy/diffpy.srfit
```

Comment:
please have them cite the diffpy-cmi complex modeling paper.   Please have them cite this same paper for:
srfit
srreal
structure
utils
...am I forgetting any? not pdfmorph/snmf/fourigui which are separate packages.